ee/modules/650-runtime-audit-engine/images/falco/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM falcosecurity/falco:0.33.1@sha256:53ade26aeb64bfd47c276bdd103709e69837bd95f1cd66b66661f84cedd37eef
```

Comment:
Why we do not copy binaries to BASE_ALPINE image as usual ?

Reply:
This is still in TODO because Falco also needs gcc of different versions. I don't think it should block reviewing process.

Reply:
ok